todo/forms.py

```diff
@@ -53,6 +53,12 @@ def clean_created_by(self):
         """Keep the existing created_by regardless of anything coming from the submitted form.
         If creating a new task, then created_by will be None, but we set it before saving."""
         return self.instance.created_by
+    
+    def clean_completed(self):
+        """There's not a 'completed' checkbox on the edit form,
```

Comment:
> There's not a 'completed' checkbox on the edit form,
But there should be! That's the bug. Let's fix the real bug, rather than introduce this workaround. 

Just add to `task_edit.html`:

`    <input type="hidden" name="completed" value="{{ form.completed.value }}" id="id_completed">
`
I just tested that locally and it works perfectly. Then you can remove this `clean` method.